external_search.py

```diff
@@ -1493,12 +1503,12 @@ def from_worklist(cls, _db, worklist, facets):
     def __init__(self, collections=None, media=None, languages=None,
                  fiction=None, audiences=None, target_age=None,
                  genre_restriction_sets=None, customlist_restriction_sets=None,
-                 facets=None, order=None, order_ascending=False
+                 facets=None,
```

Comment:
order and order_ascending were redundant since that information can be passed in to a Facets object.